### ISSUE TITLE
Remove usage of glslang and SPIRV

### ISSUE THREAD

**Issue by Kangz** [austinEng/webgpu-samples#95]
At this time both Chromium and Firefox have reasonable support for WGSL and are close to the upstream specification (except I/O for Firefox but that's coming soon?) @austinEng what do you think of removing the SPIRV path?

**Comment by austinEng** [comment]
Yes, I am planning to do this once all the samples are ported. I was blocked on something previously. I don't remember exactly, but I can check again.

**Comment by austinEng** [comment]
Ah, blocked on support for workgroup_id in WGSL. I think @jrprice is looking at crbug.com/tint/478

**Comment by jrprice** [comment]
I've just landed support for `workgroup_id` in Tint, should be in Chrome Canary in a day or two.
